site/specs/numbers.json

```diff
@@ -5040,100 +5018,91 @@
           },
           {
             "name": "tn",
-            "description": "<p><b class=\"required\">*</b> A Telephone Number (TN) that is referenced in the order</p>\n",
+            "description": "<p><b class=\"required\">*</b> A Telephone Number (TN) that is referenced in the order</p>",
```

Comment:
If this is required, then it should be changed to `required: true` and `<b class=\"required\">*</b>` should be deleted (along with the `<p>`)